src/js/components/Button/doc.js

```diff
@@ -394,6 +398,10 @@ export const themeDoc = {
     description: `The color of the label for primary buttons.`,
     type: 'string | { dark: string, light: string }',
   },
+  'button.primary.font.weight': {
+    description: `The wight of the text label for default buttons.`,
```

Comment:
```suggestion
    description: `The wight of the text label for primary buttons.`,
```

Reply:
Typo "weight" instead of "wight"

Reply:
@halocline good catch!

Reply:
Meet https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker , it is a good friend of mine :)